sphinx/source/conf.py

```diff
@@ -6,24 +6,24 @@
 # -- Project information -----------------------------------------------------
 # https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
 
-project = 'PnPQ'
-copyright = '2024, PnPQ contributors'
-author = 'PnPQ contributors'
+project = "PnPQ"
+copyright = "2024, PnPQ contributors"
+author = "PnPQ contributors"
```

Comment:
Interesting, how come it got changed to double quotes since the last PR...

Reply:
I think black likes double quotes more than single quotes, so when I ran `black .` that one time, it changed all of the quotes to double quotes. But it should be fine, right? Hopefully.

Reply:
Double and single quotes are functionally equivalent in Python. In the old days, some people used to suggest using `'` for machine-readable strings and `"` for human-readable strings, like help texts. This is impossible to enforce programatically and I think tools like black have decided to make everything double-quotes, which is fine. I'm not sure if there's a PEP that makes that guidance official.

Probably should figure out how to make sure black is running on the right directories in another PR, though.